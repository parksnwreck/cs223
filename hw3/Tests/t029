#!/bin/bash
# Long circuit

trap "/usr/bin/killall -q -u $USER Heatmap 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15

/c/cs474/bin/run -stderr=/dev/null ./Heatmap 0.003 0.002 .:xX# 10 < /c/cs223/hw3/Tests/east_rock.in
