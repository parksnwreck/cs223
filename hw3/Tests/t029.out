.........X#......
.........#X......
.........#x......
.........#.......
....Xxx####......
....x..#.###.....
....x.....###....
....#......#x....
:XX#X......x.....
x...........#.xX.
x...........XXx:x
X...............x
.X..............x
x#.............X.
#.......#......x.
#.......Xx#....X.
.X#x:...X.:...X..
...##XxX#..#X#x..
